docs/api/clipboard.md

```diff
@@ -49,14 +62,9 @@ Writes the `image` into clipboard.
 
 Clears everything in clipboard.
 
-## clipboard.has(format[, type])
```

Comment:
I think we should keep the `clipboard.has` API, most use cases tend to test specified type instead of iterating all types.